scala-package/examples/src/test/scala/org/apache/mxnetexamples/infer/objectdetector/ObjectDetectorExampleSuite.scala

```diff
@@ -35,7 +35,7 @@ class ObjectDetectorExampleSuite extends FunSuite with BeforeAndAfterAll {
     logger.info("tempDirPath: %s".format(tempDirPath))
 
     val modelBase = "https://s3.amazonaws.com/model-server/models/resnet50_ssd/"
-    val synsetBase = "https://raw.githubusercontent.com/awslabs/mxnet-model-server/master/examples/"
+    val synsetBase = "https://s3.amazonaws.com/model-server/models/resnet50_ssd/"
```

Comment:
synsetBase = modelBase ?

Reply:
Not the same.

Reply:
the value for both seem to be the same, we don't need two variables with same value.